spec/controllers/users_controller_spec.rb

```diff
@@ -1,9 +1,74 @@
 require 'rails_helper'
 
 RSpec.describe UsersController, type: :controller do
-  let(:manager) { create :manager }
+  let(:manager) { create :manager, full_name: 'John Doe' }
```

Comment:
Use `find_or_create :disclosure_bmt_user`. This will be a manager on the right team for all cases created.